src/components/BButton/BButton.vue

```diff
@@ -1,18 +1,75 @@
 <template>
-  <button>{{text}}</button>
+  <button
+    v-bind:class="computedClass"
```

Comment:
i thought we were going with shorthand?

Reply:
I apparently missed that in the conversation, will change

Reply:
Oh yeah, I think we said that while you were out of the room